private/pkg/cert/certclient/util.go

```diff
@@ -18,11 +18,12 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"errors"
+	"fmt"
 	"os"
 )
 
 // NewClientTLSConfigFromRootCertFiles creates a new tls.Config from a root certificate files.
-func NewClientTLSConfigFromRootCertFiles(rootCertFilePaths ...string) (*tls.Config, error) {
+func NewClientTLSConfigFromRootCertFiles(alsoUseSystem bool, rootCertFilePaths ...string) (*tls.Config, error) {
```

Comment:
Can you make an option `WithSystemCertPool()`

Reply:
done, ptal